src/osd/OSD.cc

```diff
@@ -2512,6 +2513,25 @@ void OSD::create_logger()
 
   PerfCountersBuilder osd_plb(cct, "osd", l_osd_first, l_osd_last);
 
+  // Latency axis configuration for op histograms, values are in nanoseconds
+  PerfHistogramCommon::axis_config_d op_hist_x_axis_config{
+    "Latency (usec)",
+    PerfHistogramCommon::SCALE_LOG2, ///< Latency in logarithmic scale
+    0,                               ///< Start at 0
+    100000,                          ///< Quantization unit is 100usec
```

Comment:
I think a fixed quantization value is **not** suitable for latency histograms. For instance, one wants to distinguish between 0.1 msec and 1 msec, but measuring latencies ~ 1 *sec* with that precision hardly makes any sense. Perhaps we should stop re-inventing the wheel and re-use [HdrHistogram](https://github.com/HdrHistogram/HdrHistogram)